holidays/countries/austria.py

```diff
@@ -19,6 +19,7 @@
 class Austria(HolidayBase, ChristianHolidays, InternationalHolidays):
     country = "AT"
     default_language = "de"
+    supported_categories = ("bank", "public")
```

Comment:
It'd be better to introduce constants for each category and use them instead of literals.